File: cookbook/node/automations/links/create.js
import { config } from 'dotenv';
import { Retab } from '@retab/node';

// Load environment variables
config({ path: '../../../.env.production' });

const reclient = new Retab();

const automation = await reclient.processors.automations.links.create({
  name: "Link Automation",
  processor_id: "proc_o4dtLxizT0kDAjeKuyVLA",
  webhook_url: "https://your-server.com/webhook",
});

console.log(JSON.stringify(automation, null, 2));

File: cookbook/python/automations/links/create.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../../.env.production")

reclient = Retab()

automation = reclient.processors.automations.links.create(
    name="Link Automation",
    processor_id="proc_o4dtLxizT0kDAjeKuyVLA",
    webhook_url="https://your-server.com/webhook",
)

print(automation.model_dump_json(indent=2))
